evm-contracts/src/v0.6/tests/CheckedMathTestHelper.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.6.0;
+
+import '../dev/CheckedMath.sol';
+
+contract CheckedMathTestHelper {
+  using CheckedMath for int256;
+
+  function add(int256 a, int256 b)
+    external
+    pure
+    returns (int256 result, bool ok)
+  {
+    return a.add(b);
+  }
+
+  function sub(int256 a, int256 b)
+    external
+    pure
+    returns (int256 result, bool ok)
+  {
+    return a.sub(b);
+  }
+
+  function mul(int256 a, int256 b)
+    external
+    pure
+    returns (int256 result, bool ok)
+  {
+    return a.mul(b);
+  }
+
+  function div(int256 a, int256 b)
+    external
+    pure
+    returns (int256 result, bool ok)
+  {
+    return a.div(b);
+  }
```

Comment:
Why do you need to do this wrapper on top of int256 methods?

Reply:
these are just for testing. The corresponding methods on CheckedMath are `internal` and therefore not callable by the tests. So we wrap them and make the wrapper `public`. This is a common pattern